coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -1007,9 +1027,12 @@ def layer_norm(context, node):
     bias = inputs[3]
     eps = inputs[4]
     # cudnn_enable = inputs[5] unused
+    axes = _get_axes_from_normalized_shape(_input.shape, normalized_shape.val)
+    axes = mb.const(val=axes, name=node.name + "_axes")
```

Comment:
Oh also no need to add the const node here ...

Reply:
You can either pass a list or bumpy array as long as they are constant :D

Reply:
Used it to assign a unique name to the axes argument, do we think it is not needed downstream?

Reply:
The `axes` argument is only used by this node I think, so there is no need to add a const node here I think 😄

Reply:
👍 👍